.gitignore

```diff
@@ -1 +1,6 @@
 /.idea/
+*.iml
+/out/production/MachinesHaveNeeds/
```

Comment:
@AraChilingaryan the same is here no need to add all separately

Reply:
done